src/pages/documentation/places/index.md

```diff
@@ -22,33 +22,17 @@ The SDK extension listens for events that contain GPS coordinates and geofence r
 
 ### Download and import the Places extension
 
-<TabsBlock orientation="horizontal" slots="heading, content" repeat="3"/>
+<TabsBlock orientation="horizontal" slots="heading, content" repeat="1"/>
 
-Android
-
-<Tabs query="platform=android&task=download"/>
-
-iOS (AEP 3.x)
-
-<Tabs query="platform=ios-aep&task=download"/>
-
-iOS (ACP 2.x)
+iOS
 
 <Tabs query="platform=ios-acp&task=download"/>
 
 ### Register the Places extension with Mobile Core
```

Comment:
Looks like the Places API reference contains details for AEPPlaces 3.x only. Since we will not preserve that page in this repo, I suggest we add a section at the end of this overview page with a link to the full API reference, e.g.:

A full API reference for the Android Places extension and iOS Places can be found [here](https://experienceleague.adobe.com/docs/places/using/places-ext-aep-sdks/places-extension/places-api-reference.html?lang=en).